### ISSUE TITLE
add `identificationVerificationStatus` field

### ISSUE THREAD

**Issue by stijnvanhoey** [inbo/data-publication#128]
In order to improve the subselection by researchers, a status field would support an estimate of the confidence of the individual records.

**Comment by peterdesmet** [comment]
Agreed. Do we have any information that could be used to populate this field?

**Comment by SanderDevisscher** [comment]
Yes we do! In the recorder front-end it is named taxon determination type.
![image](https://user-images.githubusercontent.com/16779320/54022449-cb19b100-4192-11e9-8819-ce1d4892a492.png)
